Library/Homebrew/test/Gemfile

```diff
@@ -1,9 +1,15 @@
 source "https://rubygems.org"
 
-gem "mocha", "~> 1.1"
-gem "minitest", "~> 5.3"
-gem "rake", "~> 10.3"
-gem "parallel_tests", "~> 2.9"
+group :test do
+  gem "minitest", "~> 5.9"
+  gem "minitest-reporters"
+  gem "mocha", "~> 1.2"
+  gem "rake", "~> 10.5"
+  gem "rspec", "~> 3.5.0"
```

Comment:
As mentioned before: I'm a very strong 👎 on installing MiniTest and RSpec for the unified `brew tests`. We need to pick one test suite and use that.